inc/3rd-party/hosting/presslabs.php

```diff
@@ -0,0 +1,139 @@
+<?php
+defined( 'ABSPATH' ) || die( 'Cheatin&#8217; uh?' );
+
+if ( defined( 'PL_INSTANCE_REF' ) && class_exists( '\Presslabs\Cache\CacheHandler' ) ) {
+
+	add_action( 'pl_pre_cache_refresh', 'rocket_clean_post', 0 );
+	add_action( 'after_rocket_clean_domain', 'rocket_pl_clear_cache' );
+	add_action( 'after_rocket_clean_post', 'rocket_pl_clean_post', 2 );
```

Comment:
You don't need this one, as `rocket_clean_post()` is calling `rocket_clean_files()`, so your action on line 13 will take care of cache clearing.

If  you keep it, there will be a double call to clearing the cache, which is unnecessary. You can remove the action and the associated function.

Reply:
OK, I have removed it completely and modified the one that used to be on line 13.